PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -155,12 +154,22 @@
     looseId = cms.InputTag("looseJetId"),
 )
 
+lepInJetVars = cms.EDProducer("LepInJetProducer",
+    src = cms.InputTag("updatedJetsAK8"),
+    srcEle = cms.InputTag("slimmedElectrons"),
```

Comment:
I start the test already, but this is incorrect.
Not all elements of the slimmedElectrons and slimmedMuons are saved, so the indices will be invalid once read in NanoAOD. Please adapt to use the finalElectrons and finalMuons collections.

Reply:
Ok I just made the changes to finalElectrons and finalMuons